ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/Errors.java

```diff
@@ -40,13 +40,19 @@
   public static final int ERROR_CODE_BAD_STATEMENT = toErrorCode(BAD_REQUEST.code()) + 1;
   private static final int ERROR_CODE_QUERY_ENDPOINT = toErrorCode(BAD_REQUEST.code()) + 2;
 
+  public static final int ERROR_CODE_MAX_PUSH_QUERIES_EXCEEDED =
```

Comment:
This PR is a breaking change since some of the error codes for the new API endpoints have changed. Can we mark it as a breaking change in the PR and commit description? Do we need to make updates to any of the examples that use the new API endpoints (e.g., the Javascript ride-sharing app)?

Reply:
We haven't released the new API yet, so I think it is ok :)